tensorflow_lite_support/ios/TensorFlowLiteTaskVision.podspec.template

```diff
@@ -0,0 +1,62 @@
+Pod::Spec.new do |s|
+  s.name             = 'TensorFlowLiteTaskVision'
+  s.version          = '${TFLS_BUILD_VERSION}'
+  s.authors          = 'Google Inc.'
+  s.license          = { :type => 'Apache',:file => "LICENSE.txt"}
+  s.homepage         = 'https://github.com/tensorflow/tflite-support'
+  s.source           = { :http => '${TFLS_DOWNLOAD_URL}' }
+  s.summary          = 'TensorFlow Lite Task Library - Vision'
+  s.description      = 'The Natural Language APIs of the TFLite Task Library'
+
+  s.ios.deployment_target = '13.0'
```

Comment:
@priankakariatyml Can this be reduced to 9.0 as in the Text Library and TFLite? 

iOS 13 was released in 2019, which is too recent. We want to support older devices as well.